src/admin/pages/Settings.vue

```diff
@@ -609,11 +609,11 @@
 
         div.nav-tab-wrapper {
             width: 340px;
-            border: 1px solid #c8d7e1;
+            //border: 1px solid #c8d7e1;
```

Comment:
kindly remove all the commented out code if you don't need this code anymore. You'll be able to retrieve old code vai git if needed in the future.

Reply:
Removed bhai.